cmd/kube-proxy/app/server.go

```diff
@@ -323,6 +335,20 @@ func (s *ProxyServer) Run(_ []string) error {
 		}, 5*time.Second, util.NeverStop)
 	}
 
+	// Tune conntrack, if requested
```

Comment:
not part of this change, but should s.birthCry(), line 326, be just before SyncLoop(), line 353 ?